endorse/sp800155_test.go

```diff
@@ -64,30 +64,29 @@ func TestMakeEvents(t *testing.T) {
 	if err != nil {
 		t.Fatalf("makeEvents(%v) failed: %v", e, err)
 	}
-	varEvt := combine([]byte{0x92, 0x00}, // Event length as little endian uint16
-		[]byte("SP800-155 Event3"),
+	varEvt := combine([]byte("SP800-155 Event3"),
 		binary.LittleEndian.AppendUint32(nil, 11129), // PlatformManufacturerID
 		rimEFIGUID, // ReferenceManifestGuid
-		append([]byte{byte(len(googleManufacturer))}, []byte(googleManufacturer)...), // PlatformManufacturerStr
-		append([]byte{byte(len(platformModel))}, []byte(platformModel)...),           // PlatformModel
-		[]byte{0}, // PlatformVersion
-		append([]byte{byte(len(googleManufacturer))}, []byte(googleManufacturer)...), // FirmwareManufacturerStr
-		binary.LittleEndian.AppendUint32(nil, 11129),                                 // FirmwareManufacturerID
-		[]byte{0}, // FirmwareVersion
-		binary.LittleEndian.AppendUint32(nil, eventlog.RIMLocationVariable), // RIM locator type
-		binary.LittleEndian.AppendUint32(nil, uint32(len(rimVar))),          // RIM locator (length)
+		append([]byte{byte(len(googleManufacturer) + 1)}, []byte(googleManufacturer+"\x00")...), // PlatformManufacturerStr
```

Comment:
So these are both length prefixed and null terminated?

Reply:
Yes, because TCG tries to accommodate folks who want to printf and not use string views.